basics/a27.go

```diff
@@ -0,0 +1,35 @@
+// Understanding basic variable data types
+
+package main
+
+import "fmt"
+
+func main() {
+	var a = "/home"
+	fmt.Printf("Data type of variable a is %T\n", a)
```

Comment:
Can we have 'a' in quotes?

Reply:
No we can do it  only if it's a single character . Refer: https://stackoverflow.com/questions/34691045/cannot-assign-string-with-single-quote-in-golang

Reply:
I just tried `fmt.Printf("Data type of variable 'a' is %T\n", a)` and it prints as expected `Data type of variable 'a' is string`

Reply:
Data type of variable 'a' is int32

Reply:
okay updated the code.